rocks-lib/src/lockfile/mod.rs

```diff
@@ -6,10 +6,12 @@ use itertools::Itertools;
 use serde::{Deserialize, Serialize};
 use sha2::{Digest, Sha256};
 
-use crate::lua_package::{LuaPackage, PackageName, PackageVersion, PackageVersionReq};
+use crate::lua_package::{
+    PackageName, PackageReq, PackageVersion, PackageVersionReq, RemotePackage,
+};
 
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq, PartialOrd, Ord)]
-pub struct LockedPackage {
+pub struct LuaPackage {
```

Comment:
This is the first major rename in the API: a `LuaPackage` now represents a physical package present on the filesystem, full of important metadata like dependencies and version constraints.

Reply:
Let's set add this info as a docstring.
I'm wondering, would `LocalPackage` be a more suitable name?